networking.md

```diff
@@ -0,0 +1,92 @@
+# Filecoin Network
+
+The filecoin network is built using libp2p building blocks, as transports and protocols, as well as some additional filecoin specific protocols as outlined in [Network Protocols](./network-protocols.md).
+
+## Required Protocols
+
+Every full node must support the following libp2p protocols:
+
+ - [gossipsub](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub) ([for data announcements](./data-propagation.md))
+ - [bitswap] ([for data exchange](./))
+ - Filecoin specific Protocols:
+   - Hello Handshake
+   - StorageDeal
+   - BlockSync
+
+## Transports, Streams & Encryption
+
+Connections between nodes are end-to-end-encrypted and -authenticated, thus every node has a `peerId` associated with it. Any node can also support unencrypted or unauthenticated connection request, but it is not required to do so.
```

Comment:
I would leave off the bit about supporting unencrypted/unauthenticated connections. Thats a debug mode and is not to ever be used in production.

Reply:
> I would leave off the bit about supporting unencrypted/unauthenticated connections.

Wasn't sure if that was something we intended to support - libp2p certainly does allow that in general and I wasn't sure if that might be useful for light-client scenarios or unnecessary. I am fine with dropping it.

Reply:
I think its worth taking a hardline stance on 'all connections should be encrypted and authenticated'. leaving the option open for unencrypted comms just sounds like its asking for downgrade attacks.